README.md

```diff
@@ -1,6 +1,18 @@
 # react-server monorepo [![Build Status][build-badge-img]][build-url]
 
-(see: [README](packages/react-server/README.md))
+# Monorepo?
+
+It's a thing, now.  All the cool kids are doing it, like [React](https://github.com/facebook/react/tree/master/packages), [Meteor](https://github.com/meteor/meteor/tree/devel/packages), [Ember](https://github.com/emberjs/ember.js/tree/master/packages), and [Babel](https://github.com/babel/babel/tree/master/packages).  All of the packages are versioned in lockstep, which makes it easier for us to make strong guarantees about the interoperability of the packages that work together in the react-server ecosystem.
+
+# react-server?
+The best place to start is [react-server's README](packages/react-server/README.md)), though you will likely want to read all of [react-server's docs](packages/react-server/docs).
+
+# Contributing
+
+See [the contributing guide](CONTRIBUTING.md).
 
 [build-badge-img]: https://travis-ci.org/redfin/react-server.png?branch=master
 [build-url]: https://travis-ci.org/redfin/react-server
+[![npm version](https://badge.fury.io/js/react-server.svg)](https://badge.fury.io/js/react-server)
+[![Dependency Status](https://david-dm.org/redfin/react-server.svg)](https://david-dm.org/redfin/react-server)
+[![devDependency Status](https://david-dm.org/redfin/react-server/dev-status.svg)](https://david-dm.org/redfin/react-server#info=devDependencies)
```

Comment:
Yes, though maybe useful to _also_ put version badges in the packages?

Not all packages are published with each version.